### ISSUE TITLE
AttributeError: module 'openai' has no attribute 'error

### ISSUE THREAD

**Issue by Almas-Ali** [airtai/nbdev-mkdocs#212]
OS: Linux

```bash
Traceback (most recent call last):
  File "***/.venv/bin/nbdev_mkdocs", line 5, in <module>
    from nbdev_mkdocs._cli import _app
  File "***/.venv/lib/python3.11/site-packages/nbdev_mkdocs/_cli.py", line 13, in <module>
    from docstring_gen.docstring_generator import add_docstring_to_source
  File "***/.venv/lib/python3.11/site-packages/docstring_gen/docstring_generator.py", line 154, in <module>
    openai.error.RateLimitError,
    ^^^^^^^^^^^^
AttributeError: module 'openai' has no attribute 'error
```

**Comment by harishmohanraj** [comment]
Hi,

Thanks for reporting the bug. Will take a look at it and release a new version soon.

**Comment by harishmohanraj** [comment]
Hi,

The issue has been fixed in the latest nbdev-mkdocs version (0.6.1)
